packages/ui/src/forum/components/threads/ThreadBrowser.tsx

```diff
@@ -44,10 +44,16 @@ export const ThreadBrowser = ({
           {label} {!!totalCount && <CountBadge count={totalCount} />}
         </Label>
         <ButtonsGroup>
-          <ButtonGhost size="small" square onClick={onPrevClick} disabled={currentPage <= 1}>
+          <ButtonGhost size="small" square onClick={onPrevClick} disabled={currentPage <= 1} title="Browse left">
```

Comment:
```suggestion
          <ButtonGhost size="small" square onClick={onPrevClick} disabled={currentPage <= 1} title="Browse previous">
```

Reply:
I forgot to write: Left/right is not accurate. Previous/Next is a bit better + it is safer for RTL languages as well ;)